io/net/sriov_device_test.py

```diff
@@ -67,13 +67,15 @@ def setUp(self):
         if not self.server:
             self.cancel("Managed System not got")
         self.sriov_adapter = self.params.get('sriov_adapter',
-                                             '*', default=None).split(' ')
-        self.sriov_port = self.params.get('sriov_port', '*',
+                                             default=None).split(' ')
+        self.sriov_port = self.params.get('sriov_port',
                                           default=None).split(' ')
-        self.ipaddr = self.params.get('ipaddr', '*', default="").split(' ')
-        self.netmask = self.params.get('netmasks', '*', default="").split(' ')
+        self.sriov_roce = self.params.get('sriov_roce',
+                                          default=None).split(' ')
```

Comment:
if sriov_roce param is not given from other yamls.. the test case will fail.. so please add a check to see if param in yaml

Reply:
no need to split boolean valu, also better to check if param is given else other yamls will fail if it does not have parameter

```
        if 'sriov_roce' in self.params:
            self.sriov_roce = self.params.get('sriov_roce', default=None)
```

Reply:
I have incorporated the review comments.

Reply:
ignore my previous comment, the params are not iterable so it wont work